src/components/section/AvailabilityZone.tsx

```diff
@@ -164,7 +164,7 @@ export const GroupAvailabilityZone: React.FC<GroupAvailabilityZoneProps> = ({
         (participant) =>
           !hoveredTime ||
           participant.timeSlots.some((ts) =>
-            isBetween(hoveredTime, ts.begins, ts.ends)
+            isBetweenExcludeEnd(hoveredTime, ts.begins, ts.ends)
           )
       ) ?? []
```

Comment:
There were also issues here regarding time display: see the img for ref, the time seems to be off by 1 hr for some reason
<img width="1202" alt="Screenshot 2023-09-13 at 11 09 47" src="https://github.com/ParachuteTeam/Parachute/assets/20884681/e46a7109-4d85-429e-8318-02b16a3b4aa9">

Reply:
Thank you for the reminder! Fixed.